index.html

```diff
@@ -4866,7 +4866,7 @@ <h3>Fitness for Purpose</h3>
 
   <section class="appendix informative">
     <h2>Base Context</h2>
-      The base context located at <code>https://www.w3.org/2018/credentials/v1</code> with SHA256(<code>63C0C0F374AB61F4B810BEF8A6DEE5F83C7D845992EB920CAF2505C4C99BB91C</code>) can be used to implement a local cached copy.  The aforementioned context is provided in this section for convenience.
+      The base context located at <code>https://www.w3.org/2018/credentials/v1</code> with SHA-256 digest <strong><code>63C0C0F374AB61F4B810BEF8A6DEE5F83C7D845992EB920CAF2505C4C99BB91C</code></strong> can be used to implement a local cached copy.  The aforementioned context is provided in this section for convenience.
```

Comment:
github won't let me make a PR against your fork, so here's my suggested edit, as a multi-line (fitting existing doc markup style) codeblock you can copy and paste, replacing your single-line 4869 --
```
      <p>
The base context, located at 
<code>https://www.w3.org/2018/credentials/v1</code> with SHA-256 digest 
<strong><code>63C0C0F374AB61F4B810BEF8A6DEE5F83C7D845992EB920CAF2505C4C99BB91C</code></strong>, 
can be used to implement a local cached copy.  For convenient reference,
the content of that file is also provided below.
      </p>
```